react/components/SideBar.js

```diff
@@ -55,7 +57,7 @@ export default class SideBar extends Component {
   render() {
     const { visible, onClose, showSubcategories } = this.props
 
-    const scrimClasses = classNames('vtex-menu-sidebar__scrim fixed dim bg-base--inverted top-0 z-1 w-100 vh-100 o-40', {
+    const scrimClasses = classNames('fixed dim bg-base--inverted top-0 z-1 w-100 vh-100 o-40', {
```

Comment:
Why do you removed `vtex-menu-sidebar__scrim` and did not replace it for `categoryMenu.sidebarScrim`?

Reply:
was a mistake that i have made. already fixed.